lib/theme_check/liquid_node.rb

```diff
@@ -73,6 +73,21 @@ def outer_markup
       end
     end
 
+    def outer_markup_range
+      return if variable_lookup?
+      if tag? && block?
+        start_index = block_start_start_index
+        end_index = find_block_delimiter(block_start_end_index + inner_markup.size)&.end(0)
+      elsif tag?
+        start_index = block_start_start_index
+        end_index = block_start_end_index + 1
+      else
+        start_index = outer_markup_start_index
+        end_index = outer_markup_end_index + 1
+      end
+      start_index...end_index
+    end
```

Comment:
Is `outer_markup_start_index..outer_markup_end_index` not working?

It should work for all cases. Otherwise it's those methods that are broken.

Reply:
`outer_markup_start_index..outer_markup_end_index` doesn't remove newlines while the alternative ranges that I provide in `outer_markup_range` do

Reply:
Question: What's the difference between the way the `start_index` and `end_index` are calculated in `outer_markup` and simply doing `outer_markup_start_index..outer_markup_end_index`?